library/cartridge_failover_promote.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/python
+
+import pkgutil
+
+if pkgutil.find_loader('ansible.module_utils.helpers'):
+    import ansible.module_utils.helpers as helpers
+else:
+    import module_utils.helpers as helpers
+
+argument_spec = {
+    'console_sock': {'required': True, 'type': 'str'},
+    'failover_promote_params': {'required': True, 'type': 'dict'},
+}
+
+
+def failover_promote(params):
+    console_sock = params['console_sock']
+
+    specified_replicaset_leaders = params['failover_promote_params'].get('replicaset_leaders')
+    force_inconsistency = params['failover_promote_params'].get('force_inconsistency')
+
+    if not specified_replicaset_leaders:
+        return helpers.ModuleRes(changed=False)
+
+    control_console = helpers.get_control_console(console_sock)
+
+    cluster_instances = helpers.get_cluster_instances(control_console)
+    cluster_replicasets = helpers.get_cluster_replicasets(control_console)
+
+    replicaset_leaders = dict()
+    for replicaset_alias, leader_alias in specified_replicaset_leaders.items():
+        cluster_replicaset = cluster_replicasets.get(replicaset_alias)
+        if cluster_replicaset is None:
+            return helpers.ModuleRes(
+                failed=True,
+                msg="Replicaset %s isn't found in cluster" % replicaset_alias
```

Comment:
```suggestion
                msg="Replicaset '%s' isn't found in cluster" % replicaset_alias
```

The same comment for others